### PR TITLE
util/math: migrate test-infra to testify

### LINKED ISSUE #26978: migrate test-infra to testify for `util/math` pkg

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### What problem does this PR solve?

Issue Number: close #26978 

### Release note

```release-note
None
```

### CODE REVIEW

File: util/math/main_test.go [hunk 1]
```diff
@@ -0,0 +1 @@
+package math
```
Comment (mmyj) [inline util/math/main_test.go:1]: Add a licence header.

File: util/math/math_test.go [hunk 2]
```diff
@@ -18,39 +18,34 @@ import (
...
-		^uint64(0),
-	}
-
-	for _, num := range nums {
-		expected := len(strconv.FormatUint(num, 10))
-		actual := StrLenOfUint64Fast(num)
-		c.Assert(actual, Equals, expected)
-	}
+func TestStrLenOfUint64Fast(t *testing.T) {
+	t.Parallel()
```
Comment (mmyj) [inline util/math/math_test.go:25]: Moving it to the subtests?

Comment (tisonkun) [reply util/math/math_test.go:25]: Make sense

**Comment by tisonkun** [comment]
@mmyj updated.

**Review by mmyj — APPROVED** [review]
lgtm

**Review by xhebox — APPROVED** [review]

**Pull request merged**